documentation/using-azure-identity.md

```diff
@@ -309,7 +309,7 @@ locally, you can also add a redirect URI for your development endpoint
 (e.g. `http://localhost:8080/authresponse`).
 
 A complete example of hosting your own authentication response endpoint can be
-found in the [`authorization code sample`](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/samples/manual/authorizationCodeSample.ts).
+found in the [`authorization code sample`](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/test/manual/authorization-code-credential/authorizationCodeSample.ts).
```

Comment:
Hello hello :) is this older? what about doing what we did here https://github.com/Azure/azure-sdk-for-js/pull/18887 ?

Reply:
The intent here was to eliminate the incorrect link so that I could verify that the CI would pass E2E. Once your change merged I rebased and took your change.

Reply:
ok awesome! thank you!